### ISSUE TITLE
Prevent Appbar From Being Resized With Browser Zooming

### ISSUE THREAD

**Issue by TheRealDannyyy** [Armaldio/ttv-desktop-lite#23]
This requires some JS hacking but I'm sure you can figure it out.
(https://stackoverflow.com/questions/15233076/prevent-that-a-fixed-element-resizes-when-zooming-on-touchscreen)

**Comment by TheRealDannyyy** [comment]
If you can't fix this, maybe blocking _shortcut zooming_ (ctrl + wheel ...) and adding "zoom in, zoom out, zoom 100%" in the settings menu could be a workaround.

**Comment by Armaldio** [comment]
I already cannot zoom in current version can you confirm ?

**Comment by TheRealDannyyy** [comment]
Weird I remember being able to zoom in past versions. Nope cannot zoom either.

**Comment by Armaldio** [comment]
Update the issue according to the desired behaviour then

**Comment by TheRealDannyyy** [comment]
Honesty up to you. Do what's easier.

**Comment by TheRealDannyyy** [comment]
Yes I want resizing but the method is up to you.

**Comment by TheRealDannyyy** [comment]
@Armaldio I've added a sublabel so that users can see the current zoom level, could you please fetch the value and replace the placeholder text?

Everything else works great.

**Comment by TheRealDannyyy** [comment]
I've check this out again and pressing Ctrl- - breaks the top bar and needs a manual workaround.
You have to replace the following shortcuts with custom webview zooming:
- Ctrl & + = Zoom in 
- Ctrl & - = Zoom out
- Ctrl & 0 = Set Zoom to 100%

**Issue blocked by:**
- [ ] Custom shortcuts for zooming
- [ ] Current Zoom amount display in menu
- [ ] Allow Fullscreen zooming using custom shortcuts

**Comment by Armaldio** [comment]
Seems fixed

**Comment by TheRealDannyyy** [comment]
I guess you decided to entirely block shortcuts and make it settings menu only. No problem with that so closing.

**Comment by Armaldio** [comment]
No, shortcuts are working for me: https://github.com/Armaldio/ttv-desktop-lite/commit/4eafff7e09eef668894e37a22854c95ebc11c7e5#diff-bd086afd87b637879590bdc309e5b891R387

- Ctrl & + = Zoom in
- Ctrl & - = Zoom out
- Ctrl & 0 = Set Zoom to 100%

**Comment by TheRealDannyyy** [comment]
You are right. They work but Ctrl & + isn't for some reason

**Comment by Armaldio** [comment]
🤔 Might be a keyboard layout related problem, will check, keep this openened

**Comment by TheRealDannyyy** [comment]
Maybe adding an OR with Crtl + mousewheel up and Crtl + mousewheel down could be good enough as a workaround for special keybord layouts.

The zoom level also resets on restart, it should be saved and applied on pageload (do not apply to Twitchy loading page).

**Comment by TheRealDannyyy** [comment]
I can also still reproduce the appbar zooming using shortcuts. I would honestly recommend to just removing shortcuts and handle zooming entirely using the settings menu.
